### PR TITLE
Perf stress_ng test fixes

### PR DESCRIPTION
**Pull request by kjain101**
Made changes in perf/perf_stress_ng.py test to fix vmstat calculation and change percentage calculation part.
Also add code to find system CPU frequency for calculations.

### CODE REVIEW

**Review by disgoel — APPROVED** [review]
test works fine with the changes

> [root@ltcden9-lp2 perf]# avocado run perf_stress_ng.py -m perf_stress_ng.py.data/perf_stress_ng.yaml --max-parallel-tasks=1
Fetching asset from perf_stress_ng.py:Stressng.test
JOB ID     : 76995f615712ab51902b534f5edbd362fc9c4a2e
JOB LOG    : /home/avocado-fvt-wrapper/results/job-2023-08-07T08.05-76995f6/job.log
 (1/1) perf_stress_ng.py:Stressng.test;run-coverage-run_type-df37: STARTED
 (1/1) perf_stress_ng.py:Stressng.test;run-coverage-run_type-df37: WARN: Test passed but there were warnings during execution. Check the log for details. (199.64 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 1 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/avocado-fvt-wrapper/results/job-2023-08-07T08.05-76995f6/results.html
JOB TIME   : 202.50 s

**Comment by PraveenPenguin** [comment]
@kjain101 PR looks good to me can you please rebase this PR as it has merge conflict

**Comment by kjain101** [comment]
Hi Praveen, I fixed the merge conflicts.

File: perf/perf_stress_ng.py [hunk 1]
```diff
@@ -130,6 +131,8 @@ def test(self):
         smt = int(re.split(r'=| is ', process.system_output("ppc64_cpu --smt")
                            .decode('utf-8'))[1])
         no_of_cores = int(self.tcpus // smt)
+        clock_freq = int(subprocess.check_output("cat /proc/cpuinfo | grep -m1 clock |sed 's/.*://' | cut -f1 -d'.'", shell=True))
```
Comment (Naresh-ibm) [inline perf/perf_stress_ng.py:134]: @kjain101 can we use like below  
```
cmd = "cat /proc/cpuinfo | grep -m1 clock |sed 's/.*://' | cut -f1 -d'.'"
 clock_freq = int(subprocess.check_output(cmd, shell=True)
```

Comment (kjain101) [reply perf/perf_stress_ng.py:134]: Hey naresh yes that looks fine to me. I will do the changes

**Comment by kjain101** [comment]
I will send new pull request with suggested changes. Closing this one.

**Pull request closed without merging**